src/utilities/theme/role-variants.ts

```diff
@@ -110,6 +110,22 @@ export const roleVariants: RoleVariants = {
         alpha: 0.15,
       },
     },
+    {
+      name: 'hintFromDirectLight',
+      description: 'For use in building shadows scrollables.',
+      light: {
+        hue: 0,
+        saturation: 0,
+        lightness: 0,
+        alpha: 0.15,
+      },
+      dark: {
+        hue: 0,
+        saturation: 0,
+        lightness: 75,
+        alpha: 0.2,
+      },
+    },
```

Comment:
Since these are the same as the existing shadows, we don't need to add new ones. Use `shadowFromDirectLight` instead.

Adding new ones just for this use feels like premature optimization, since we have no indication that these will ever be different from existing shadows.

Reply:
They're slightly different. Between light & dark, `shadowFromDirectLight` has the same HSL values. `hintFromDirectLight` has different HSL values for dark, so the hint prop on scrollable will work in dark mode.

Reply:
My bad, didn't notice the difference when I first looked!